layers/dns_layer.py

```diff
@@ -164,7 +164,7 @@ def dns_decompress(packet, logger):
         # Next, we must rebuild the DNS packet itself. If we try to have scapy parse either dns_qr1 or dns_qr2, they
         # will look malformed, since neither contains a complete request. Therefore, we must build the entire
         # DNS packet at once. First, we must remove the original DNSQR, since this contains the original request
-        del packet["DNS"].qd
+        packet["DNS"].qd = None
```

Comment:
Can you add a comment here as to why this change was necessary? (scapy 2.5.0 compat)